rotkehlchen/db/filtering.py

```diff
@@ -311,6 +311,7 @@ def make(
             tx_hash: Optional[EVMTxHash] = None,
             protocols: Optional[List[str]] = None,  # Events data
             asset: Optional[Asset] = None,  # Events data
+            exclude_ignored_assets: bool = False,  # Events data
```

Comment:
Aha I see, so you wanna do it like that. Not a bad idea. Initially I had thought of a more complicated idea of introducing a NOT IN assets filter ... but yeah this is better probably. As the frontend won't have a "NOT IN" assets filter. It will just say assets to filter for .. and the NOT IN should be for ignored assets.